slipcover/bytecode.py

```diff
@@ -466,9 +469,15 @@ def add_const(self, value):
         return len(self.consts)-1
 
 
-    def insert_function_call(self, offset, function, args):
-        """Inserts a function call."""
+    def insert_function_call(self, offset, function, args, repl_length=0):
+        """Inserts a function call.
 
+        *repl_length*, if passed, indicates the number of bytes to replace at
+        that offset.  The inserted bytecode must currently be at least as
+        large as *repl_length*.
```

Comment:
So really, shouldn't it be `repl_min_length`?

Reply:
No... the intention is that you pass in `repl_length` asking it to replace that many bytes. The 2nd sentence was just documenting a current limitation.